src/socketio/asyncio_aiopika_manager.py

```diff
@@ -47,17 +47,28 @@ def __init__(self, url='amqp://guest:guest@localhost:5672//',
         self.listener_connection = None
         self.listener_channel = None
         self.listener_queue = None
+
+        self.aiopika_connection = None
+        self.aiopika_channel = None
+        self.aiopika_exchange = None
+
         super().__init__(channel=channel, write_only=write_only, logger=logger)
 
     async def _connection(self):
-        return await aio_pika.connect_robust(self.url)
+        if (self.aiopika_connection is None or self.aiopika_connection.is_closed is True):
+            return await aio_pika.connect_robust(self.url)
+        return self.aiopika_connection
```

Comment:
This should have a lock around it as if there are several calls in close succession, it'll create several new connections. See #1142 for an example of this in use.